roles/matrix-bot-maubot/templates/config/config.yaml.j2

```diff
@@ -64,7 +64,7 @@ server:
     appservice_base_path: /_matrix/app/v1
     # The shared secret to sign API access tokens.
     # Set to "generate" to generate and save a new token at startup.
-    unshared_secret: {{ matrix_bot_maubot_secret }}
+    unshared_secret: {{ matrix_bot_maubot_secret|to_json }}
```

Comment:
Calling this variable `matrix_bot_maubot_server_unshared_secret` will be even better.

Where appropriate, we try to name variables after the configuration key they control.

Similarly, it's probably better if `matrix_bot_maubot_bot_server_public` is called `matrix_bot_maubot_bot_server_public_url`.

Reply:
done